src/components/StatsBar/index.tsx

```diff
@@ -0,0 +1,119 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { IconProp } from '@fortawesome/fontawesome-svg-core';
+import range from 'lodash/range';
+import React from 'react';
+import { Box } from '../Box';
+import { Flex, FlexProps } from '../Flex';
+import { ProgressBar, ProgressBarProps } from '../ProgressBar';
+import { Txt } from '../Txt';
+
+export interface StatsBarProps extends Omit<FlexProps, 'title'> {
+	title: React.ReactElement;
+	statsLabel: React.ReactElement;
```

Comment:
How about having a labelFormatter prop instead, to which we pass `value` and `max`? Like that the caller doesn't have to keep track of those two in two separate places. Since this is a read-only component it doesn't matter as much, but IMO it is a bit cleaner

Reply:
@sradevski ok - I've made that change. Let me know if you think it's suitable or if I've missed something. (Apologies for the force-push).